logger.js

```diff
@@ -12,33 +12,45 @@
  *
  */
 
-import { createLogger, format, transports } from 'winston'
+import * as winston from 'winston'
+// import { createLogger, format, transports } from 'winston'
```

Comment:
Commented code is not ideal, I dunno what this one is about. But apparently Winston is a 3rd party to help with logging. I never heard about it, and it's modifying some things below, I have no experience. https://www.npmjs.com/package/winston